scenes/passy_scene_read_success.c

```diff
@@ -41,15 +41,15 @@ void passy_scene_read_success_on_enter(void* context) {
             }
 
             if(dg1->mrz.buf[0] == 'I' && dg1->mrz.buf[1] == 'P') {
-                furi_string_cat_printf(str, "Passport card\n");
+                furi_string_cat_printf(str, "Passport card (IP)\n");
```

Comment:
I'm not sure how much we get by adding these: most people don't know/care about the internal designation.  Also, this is another thing that is independent of the PACE stuff and could be it's own PR

Reply:
I think it's the internal intricacies that make people want to figure out how stuff works and thus showing these details is helpful. A separate PR sounds good.